### ISSUE TITLE
ListView ContextActions cause app to crash on Android

### ISSUE THREAD

**Issue by dawidpieper** [dotnet/maui#6647]
### Description

On Android, When opening context menu of a ListView with a long press gesture, the app crashes. This behavior is observed with screenreader turned on, I cannot confirm if this issue occurs without assistive technologies.

### Steps to Reproduce

1. Create MAUI Project.
2. Create a ListView with any ContextActions.
3. Run app and long press any item on the ListView.
4. The app will crash.

### Version with bug

Release Candidate 2 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 11

### Did you find any workaround?

Not found, except writing own Context Menu with long press custom renderer.

### Relevant log output

```shell
Error: Unhandled Exception
System.Exception: CurrentDomainOnUnhandledException
 ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.Maui.Controls.Handlers.Compatibility.CellAdapter.CreateContextMenu(IMenu menu)
   at Microsoft.Maui.Controls.Handlers.Compatibility.CellAdapter.OnCreateActionMode(ActionMode mode, IMenu menu)
   at Android.Views.ActionMode.ICallbackInvoker.n_OnCreateActionMode_Landroid_view_ActionMode_Landroid_view_Menu_(IntPtr
 jnienv, IntPtr native__this, IntPtr native_mode, IntPtr native_menu)
   at Android.Runtime.JNINativeWrapper.Wrap_JniMarshal_PPLL_Z(_JniMarshal_PPLL_Z callback, IntPtr jnienv, IntPtr klazz,
IntPtr p0, IntPtr p1)
--- End of stack trace from previous location ---
   at Java.Interop.JniEnvironment.InstanceMethods.CallNonvirtualObjectMethod(JniObjectReference instance, JniObjectRefer
ence type, JniMethodInfo method, JniArgumentValue* args)
   at Java.Interop.JniPeerMembers.JniInstanceMethods.InvokeVirtualObjectMethod(String encodedMember, IJavaPeerable self,
 JniArgumentValue* parameters)
   at Android.App.Activity.StartActionMode(ICallback callback)
   at Microsoft.Maui.Controls.Handlers.Compatibility.CellAdapter.HandleContextMode(View view, Int32 position)
   at Microsoft.Maui.Controls.Handlers.Compatibility.CellAdapter.OnItemLongClick(AdapterView parent, View view, Int32 po
sition, Int64 id)
   at Android.Widget.AdapterView.IOnItemLongClickListenerInvoker.n_OnItemLongClick_Landroid_widget_AdapterView_Landroid_
view_View_IJ(IntPtr jnienv, IntPtr native__this, IntPtr native_parent, IntPtr native_view, Int32 position, Int64 id)
   --- End of inner exception stack trace ---
```

**Comment by dawidpieper** [comment]
Here example code that I was using to test it.

TestPage.xaml
```
<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             x:Class="TestApp.TestPage"
             BackgroundColor="{DynamicResource SecondaryColor}">

<ScrollView>
<StackLayout>
<ListView x:Name="SWL">
<ListView.ItemTemplate>
<DataTemplate>
<ViewCell>
<ViewCell.ContextActions>
<MenuItem Text="ABC" />
</ViewCell.ContextActions>
<Label Text="{Binding .}" />
</ViewCell>
</DataTemplate>
</ListView.ItemTemplate>
</ListView>
</StackLayout>
</ScrollView>
</ContentPage>
```

TestPage.xaml.cs
```
using System;
using System.Threading.Tasks;
using System.Collections.Generic;
using Microsoft.Maui.Controls;
using Microsoft.Maui.ApplicationModel;

namespace TestApp {
	public partial class TestPage : ContentPage {
public List<string> a {get;set;}
		public TestPage() {
			InitializeComponent();
a=new List<string>();
a.Add("A");
a.Add("B");
a.Add("C");
a.Add("D");
a.Add("E");
SWL.ItemsSource = a;
}
}
}
```

**Comment by shimingsg** [comment]
verified repro on Android 12 with VS 17.3.0 Preview 1.0 [32427.505.main]

**Comment by PureWeen** [comment]
https://github.com/dotnet/maui/pull/6491
